README.md

```diff
@@ -56,6 +56,9 @@ $ pip install csvw
 - Low level CSV parsing is delegated to the `csv` module in Python's standard library. Thus, if a `commentPrefix`
   is specified in a `Dialect` instance, this will lead to skipping rows where the first value starts
   with `commentPrefix`, even if the value was quoted.
+  Also, cell content containing `escapechar` may not be round-tripped as expected,
```

Comment:
Maybe something like 'when using `escapechar` ...'?

Round-trip works within Python versions, but the output differs across Python versions, right (maybe we can be more explicit about the latter, or maybe not needed because we link to the bug)?

Reply:
As per my comment below, I don't even know what "round-trip" means exactly. Round-trip with the exact same settings? Does that make sense at all?
I guess, on the bright side, there's combinations of settings for all python versions to read all CSV correctly - but there's seems to be no automatic way to check the correctness within python :)